### PR TITLE
glfs: disable the fencing when it is not support

### PR DESCRIPTION
**Pull request by lxbsz**
According to the agreement before the fencing patch should be released since 7.6.6, but it seems not. And also in some other OS releases.

Signed-off-by: Xiubo Li <xiubli@redhat.com>

### CODE REVIEW

File: glfs.c [hunk 1]
```diff
@@ -614,6 +619,10 @@ static void glfs_async_cbk(glfs_fd_t *fd, ssize_t ret, void *data)
 	struct tcmur_cmd *tcmur_cmd = cookie->tcmur_cmd;
 	size_t length = cookie->length;
 	int err = -errno;
+	static int i = 0;
+
+	if (i++ >200)
+		return;
```
Comment (mikechristie) [inline glfs.c]: I'm not sure I understood this part. Was this some debug/test code?

File: glfs.c [hunk 2]
```diff
@@ -472,7 +473,7 @@ static glfs_t* tcmu_create_glfs_object(struct tcmu_device *dev,
 	}
 
 	ret = glfs_init(fs);
-	if (ret) {
+	if (!ret) {
```
Comment (mikechristie) [inline glfs.c]: Was this some testing code that you meant to delete, or how has it worked before?

File: glfs.c (hunk 1 above)
Comment (lxbsz) [reply glfs.c]: Sorry, this is the test code, will delete it.

File: glfs.c (hunk 2 above)
Comment (lxbsz) [reply glfs.c]: This too.

**Comment by mikechristie** [comment]
Looks ok to me. Let me know if you are waiting for any gluster team reviews or if it's ok for merging.

**Comment by lxbsz** [comment]
> Looks ok to me. Let me know if you are waiting for any gluster team reviews or if it's ok for merging.

This is reported by @pkalever,  let's wait for his reviewing he is now busying for the vagrant stuff.

@pkalever 
Could help test this please ? 
Thanks

**Comment by pkalever** [comment]
@lxbsz 
Please fix:

```
[root@server1 tcmu-runner]# make
[ 28%] Built target tcmu
[ 34%] Built target consumer
[ 37%] Built target handler_file
[ 40%] Building C object CMakeFiles/handler_glfs.dir/glfs.c.o
/root/tcmu-runner/glfs.c: In function ‘glfs_async_cbk’:
/root/tcmu-runner/glfs.c:650:4: error: a label can only be part of a statement and a declaration is not a statement
    struct glfs_state *gfsp = tcmur_dev_get_private(dev);
    ^
make[2]: *** [CMakeFiles/handler_glfs.dir/glfs.c.o] Error 1
make[1]: *** [CMakeFiles/handler_glfs.dir/all] Error 2
make: *** [all] Error 2
[root@server1 tcmu-runner]# 
```

Thanks!

**Pull request merged**